source/documentation/services/airflow/index.md

```diff
@@ -189,37 +211,41 @@ This is done using the `dag.compute_profile` key, and by default (if not specifi
 - `1vcpu`: 1 vCPU is guaranteed
 - `4gb`: 4GB of memory is guaranteed
 
-In addition to the `general` fleet, we also offer `gpu`, which provides your workflow with an NVIDIA GPU.
+In addition to the `general` fleet, we also offer `gpu`, which provides your workflow with an NVIDIA GPU pre-installed with CUDA.
 
 The full list of available compute profiles can be found [here](https://github.com/ministryofjustice/analytical-platform-airflow/blob/main/scripts/workflow_schema_validation/schema.json#L14-L41).
 
+> Analytical Platform tooling (such as JupyterLab, VSCode and RStudio) tend to run on a profile similar to `general-spot-2vcpu-8gb` or `general-spot-4vcpu-16gb`; therefore, the default compute profile is smaller.
+
 ### Multi-task
 
+![](images/airflow/airflow_diagram_deps.png)
```

Comment:
Another image, like this:

![airflow_diagram_deps](https://github.com/user-attachments/assets/290c24f2-cb4c-4a6a-b055-1c65d3de859a)